go.mod

```diff
@@ -169,6 +169,8 @@ require (
 	modernc.org/sqlite v1.17.3
 )
 
+require github.com/google/licensecheck v0.3.1
```

Comment:
can you move this up to the require that lists all our direct dependencies so we don't have yet another require section?